### ISSUE TITLE
Notify of new message in thread even if message not sent to parent channel

### ISSUE THREAD

**Issue by mkaito** [wee-slack/wee-slack#535]
Right now, when you receive a new message in a thread, but this message was not sent to parent channel too (little checkbox in Slack interface), there is no sign anywhere in the weechat interface that a new message was received, or which thread it was in. This is especially annoying if the message that originated the thread is out of sight in the scrollback buffer.

**Comment by mkaito** [comment]
In fact, I seem to get no notifications regardless of whether the thread message is sent to parent channel.

**Comment by jalcine** [comment]
This actually had me miss out on some real conversations (since we use threads a lot at work).

**Comment by trygveaa** [comment]
Notification of thread messages is now implemented with #617.

Note that the notification currently only works when the channel history is loaded. This is hopefully something we can improve in the future. I've opened #619 to track that.

Regarding when the message is sent to the parent channel too. This doesn't work in wee-slack. They are treated just like other thread messages, so they are printed in the thread buffer and you will now get notifications for them too, but they are not printed in the parent channel. I've opened #620 to track that.

If there is anything else missing, or something that doesn't work like it should, feel free to comment on it.
